.github/workflows/build.yml

```diff
@@ -0,0 +1,408 @@
+name: Build and Deploy
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [assigned, opened, synchronize, reopened]
```

Comment:
Do you need "assigned" here? If you remove `types` it will use default: [opened, synchronize, reopened]

Reply:
Agreed